bin/test_module

```diff
@@ -56,15 +56,33 @@ docker_args+=" -v $module_dir/scans:/scans"
 echo Capturing fake startup.pcap...
 sudo tcpdump -c 3 -eni $faux_intf -Z root -w $module_dir/scans/startup.pcap || true
 
+echo Staring module pcap... 
+sudo tcpdump -U -eni $faux_intf -Z root -w $module_dir/scans/test_$module.pcap &
+sleep 5
+echo Module pcap started
+
 docker_args+=" --env TARGET_IP=$faux_ip"
 docker_args+=" --env TARGET_MAC=01:02:03:04:05:06"
 
+
 mkdir -p $config_dir
 docker_args+=" -v $config_dir:/config/inst"
 docker_args+=" -v $config_dir:/config/port"
 docker_args+=" -v $config_dir:/config/device"
 docker_args+=" -v $config_dir:/config/type"
 
+# If testing the tls module, we need a few
+# additional things setup
+if [[ "$module" == "tls" ]]; then
+  #CA file is neededto validate the certs properly so we need to
+  #extract it from the faux device
+  CA_FILE=$(docker exec daq-$faux_intf cat ./tlsfaux/certs/CA_Faux.pem)
```

Comment:
we shouldn't be extracting this from the docker image... If it's needed elsewhere then we should create it during the build step and then copy it into the faux file and/or this place.  Or, in this case (testing) we can likely just generate the keys and then have them in the source tree because it's only used internally for testing.

Reply:
The certs are generated inside the faux container as it's started and are not available outside of that container.  If the cert generation is brought higher in the stack outside of the container start process, then that would require a significant amount of work and testing to verify all current tls parameters handle this new structure.  If necessary then we need to start that conversation of how that needs to look.

Reply:
I've had a think through this and I agree that extracting from the container is the lesser of all evils. I was trying out a refactoring and the problem quickly becomes one of distribution. Still doesn't make me happy, but less unhappy than alternatives.

Reply:
Agreed. Maybe an update for a later date since it's a pretty tall order.